config.yaml

```diff
@@ -49,23 +83,27 @@ collate:
   mem: 30GB
   ncpus: 4
   queue: normal
-  exe: /g/data/ik11/inputs/access-om2/bin/mppnccombine
+  exe: /g/data/vk83/apps/mppnccombine-fast/0.2/bin/mppnccombine-fast
```

Comment:
We haven't typically used `mppnccombine-fast` on the 1 degree configs, but use the `mppnccombine` executable that is built as part of the MOM model. 

I think we didn't use the fast version because it wasn't required, but I'm curious if you tried this and what was the result? I'm wondering if we should just just the fast version consistently for all models.

Reply:
Now that I look more closely, the collation step was actually failing using `mppnccombine-fast` since I need to add 
```
collate:
  mpi: true
```
I just did a test run comparing and `mppnccombine-fast` and `mppnccombine` (from MOM build). `mppnccombine-fast` was about 10x slower (2 mins vs 10 sec, though I only ran each once). The collated outputs are also different... I'm investigating further.

It looks like none of the ACCESS-NRI OM2 releases are using `mppnccombine-fast` yet, even the 01deg configs.

Reply:
> The collated outputs are also different... I'm investigating further.

This is due to `mppnccombine` outputting `NC_FORMAT_NETCDF4_CLASSIC` and `mppnccombine-fast` outputting `NC_FORMAT_NETCDF4`

Reply:
That none of the high res versions are using `mppnccombine-fast` is an oversight that needs to be fixed. It wasn't available until recently (https://github.com/ACCESS-NRI/access-om2-configs/issues/27#issuecomment-1994099458)

If `-fast` isn't, then we shouldn't use it.

Reply:
> If -fast isn't, then we shouldn't use it.

Unless there's a benefit to all the model output being `NETCDF4`, rather than having a mix? I'm not sure what this would be... I only tested on 1 month of output, but neither were _that_ slow: another test I did gave 23 sec (`mppnccombine`) vs 3 min 14 sec (`mppnccombine-fast`)

Reply:
COSIMA didn't use `-fast` so I'm cool with us not using it either if there are no benefits.